frontend/packages/kubevirt-plugin/src/types/api/V1ResourceRequirements.ts

```diff
@@ -31,8 +31,8 @@ export interface V1ResourceRequirements {
   overcommitGuestOverhead?: boolean;
   /**
    * Requests is a description of the initial vmi resources. Valid resource keys are \"memory\" and \"cpu\". +optional
-   * @type {object}
+   * @type {any}
    * @memberof V1ResourceRequirements
    */
-  requests?: object;
+  requests?: any;
```

Comment:
can it be something else?

Reply:
https://github.com/openshift/console/pull/10630 - once this will be merged I can change it back to `object`

Reply:
cool, let me know when to lgtm